QUICKSTART.md

```diff
@@ -207,10 +216,11 @@ HttpTextFormat.Getter<HttpExchange> getter =
 };
 ...
 public void handle(HttpExchange he) {
-    // Extract the context from the request
-    SpanContext ctx = tracer.getHttpTextFormat().extract(he, getter);
+  // Extract the SpanContext and other elements from the request.
+  Context ctx = tracer.getHttpTextFormat().extract(Context.current(), he, getter);
```

Comment:
`he`/`ctx` are too Go like for the Java world :) Probably we should use better variable names.

Reply:
Done.